controllers/host/host_controller.go

```diff
@@ -819,11 +819,14 @@ func (r *HostReconciler) ReconcileDisabledHost(client *gophercloud.ServiceClient
 			return err
 		}
 
-		err = r.ReconcileMemory(client, instance, profile, host)
-		if err != nil {
-			return err
-		}
+	}
```

Comment:
My question to this review is mainly:
As described in line 814: the following configuration is only expected to nodes with worker function(not sure any further development after the initial code submission).

If the memory can be configured, I don't understand why the processors and kernels cannot be configured.

Request:
1. Clarify those configurations and their update conditions
2. Need to understand if those can be applied on all the personalities: controller, worker, storage